### DISCUSSION TITLE
Can I add actionable components in a table row?

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#869] [Q&A] [answered]
I was wondering if I can add additional components such as radio button or select in a table row.

My use case is...... As a Twilio customer I've getting too many emails on error code 1120, my team member handles this error so I'd like to turn off the notification for now. Currently I don't see any option in the webhook and email trigger page. 

Thanks!

**Comment by SiTaggart** [comment, marked as answer]
It's probably my unfamiliarity with that part of the product, but I'm not able to connect the use case to actions in tables right now. You might need to provide a screenshot or something more to help us with that.

General table guidance is that in the long run the table should only be used for static, tabular data. That means, no actionable content.

This is because if the table is reasonably long, say 50 rows, with multiple actions per row, say 4, a keyboard only user would have to press the tab key 200 times to get past or beyond that table.

Because of that, there will be a special, editable/actionable Data Grid component that comes with a keyboard interaction model prevents that from happening. The entire grid is a single tab stop, and you use your arrow keys to navigate the rows and columns. See [this proof of concept](https://codesandbox.io/s/tables-41kpr?file=/src/index.tsx) for an example.

In the meantime, given that we don't have a Data Grid yet, no one is going to stop anyone putting actions in a static table. It just won't be the best UX, and you should should plan to update it once it's available.

**Reply by sang-yeo** [reply]
Thanks for the answer. We are thinking about adding an actionable content as I mentioned above and here is a quick mock for your reference. 
<img width="969" alt="Screen Shot 2020-10-19 at 5 09 45 PM" src="https://user-images.githubusercontent.com/67388401/96525003-9c98fe80-122e-11eb-87c6-28851272666d.png">
